metricbeat/metricbeat.reference.yml

```diff
@@ -383,10 +383,10 @@ metricbeat.modules:
 #-------------------------------- Munin Module -------------------------------
 - module: munin
   metricsets: ["node"]
-  enabled: false
+  enabled: true
   period: 10s
   hosts: ["localhost:4949"]
-  node.namespace: node
+  #node.namespace: node
```

Comment:
`node.namespace` is required

Reply:
Updated

This is one of these fields where I'm not sure if we should set it or not. In general we have the defaults in the reference file but already with the `hosts` part we break this principle. It seems more the reference file is going in the direction of "if you copy this, you have something running" which I kind of like.